mkdocs.yml

```diff
@@ -77,6 +80,7 @@ nav:
       - IDBAC Instrument: api/instruments/idbac_instrument.md
   - Cookbook:
     - cookbook/index.md
+    - Managing Personally Identifiable Information: cookbook/pii_tutorial.md
```

Comment:
I would give it a more use case-specific name as "PII in an LLM-powered IT Service Desk" as "Managing PII" is a bit too generic, wdyt @vincenzopecorella?

Reply:
Yes agree!